Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
I confirmed that it does fail gracefully when Internet access isn't available. Implementing in Python makes sense, but I'll wait to do anything until we make a decision. Currently I also have some hard-coded throwaway credentials for pcapr, which isn't ideal.